pkgs/development/python-modules/python-engineio/3.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, aiohttp
+, eventlet
+, iana-etc
+, libredirect
+, mock
+, requests
+, six
```

Comment:
```suggestion
```

Reply:
Actually this should be a propagatedBuildInput, fixed, thanks.